1.18/Dockerfile.rhel7

```diff
@@ -79,14 +86,18 @@ RUN sed -i -f ${NGINX_APP_ROOT}/nginxconf.sed ${NGINX_CONF_PATH} && \
     ln -s ${NGINX_LOG_PATH} /var/log/nginx && \
     ln -s /etc/opt/rh/rh-nginx${NGINX_SHORT_VER}/nginx /etc/nginx && \
     ln -s /opt/rh/rh-nginx${NGINX_SHORT_VER}/root/usr/share/nginx /usr/share/nginx && \
-    chmod -R a+rwx ${NGINX_APP_ROOT}/etc && \
-    chmod -R a+rwx /var/opt/rh/rh-nginx${NGINX_SHORT_VER} && \
-    chmod -R a+rwx ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
-    chown -R 1001:0 ${NGINX_APP_ROOT} && \
-    chown -R 1001:0 /var/opt/rh/rh-nginx${NGINX_SHORT_VER} && \
+    chown -R 1001:0 ${NGINX_CONF_PATH} && \
+    chown -R 1001:0 ${NGINX_APP_ROOT}/etc && \
+    chown -R 1001:0 ${NGINX_APP_ROOT}/src/nginx-start/  && \
     chown -R 1001:0 ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
-    chmod -R a+rwx /var/run && \
-    chown -R 1001:0 /var/run && \
+    chown -R 1001:0 /var/opt/rh/rh-nginx${NGINX_SHORT_VER} && \
+    chown -R 1001:0 /run && \
+    chmod -R ug+rwX /run && \
+    chmod -R ug+rwX /var/opt/rh/rh-nginx${NGINX_SHORT_VER} && \
+    chmod    ug+rw  ${NGINX_CONF_PATH} && \
+    chmod -R ug+rwX ${NGINX_APP_ROOT}/etc && \
```

Comment:
`chmod -R ug+rwX` means add execute only if it is directory-based on https://ss64.com/bash/chmod.html. Is it intended? If `+x` is not used, then I guess, you can not go into the directory, right?

Reply:
+x mean access also directories. `Execute (or access for directories)`

Reply:
The same for others and different versions.